### PR TITLE
j nix package

### PR DESCRIPTION
**Pull request by Hydrazer**
ok i think chose the right branch this time

### CODE REVIEW

**Comment by Hydrazer** [comment]
does someone know how to make more tests?

File: runtimes/j.nix [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            files = {
+                "test.ijs" = ''
+                    echo'OK'
+                '';
+            };
+            args = [];
+            stdin = "";
+            packages = [];
+            main = "test.ijs";
+        })
```
Comment (HexF) [inline runtimes/j.nix:67]:
```suggestion
          (piston.mkTest {
            files = {
                "test.ijs" = ''
                    echo'OK'
                '';
            };
            args = [];
            stdin = "";
            packages = [];
            main = "test.ijs";
        })
```

You can add more tests like this

File: runtimes/j.nix [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    # This is the lines of a shell script to compile source code.
+    # Arguments passed to this script are all the provided source files
+    # The CWD of this script is a temp directory for a job
+    #
+    # If the language only supports JIT compiling, simply remove this line
+    # See ./python3.nix and ./node-javascript.nix for examples
+    #
+    # No shebang needs to be added to this - that is done automatically.
+    # compile = ''
+    # ${pkg}/bin/j --compile "$@"
```
Comment (HexF) [inline runtimes/j.nix:36]: If compiling is supported, I would rather we use it.

File: runtimes/j.nix [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+    # Name of the runtime
+    # This line should be kept if the runtime differs from the language
+    # runtime = "j";
+
+    aliases = [
+        # Put extensions in here, and other common names
+        # Example:
+        # "js"
+        # "node-javascript"
```
Comment (HexF) [inline runtimes/j.nix:21]: All these comments can be removed, as can all the others from the scaffold.

**Review by HexF — APPROVED** [review]

**Review by HexF — APPROVED** [review]

**Pull request merged**